src/explore-education-statistics-admin/src/pages/release/pre-release/components/PreReleaseUserAccessForm.tsx

```diff
@@ -14,30 +16,36 @@ import useToggle from '@common/hooks/useToggle';
 import { mapFieldErrors } from '@common/validation/serverValidations';
 import Yup from '@common/validation/yup';
 import { Formik } from 'formik';
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 
 interface FormValues {
-  email: string;
+  emails: string;
 }
 
 const errorMappings = [
   mapFieldErrors<FormValues>({
-    target: 'email',
+    target: 'emails',
     messages: {
-      USER_ALREADY_EXISTS: 'User with this email already exists',
+      INVALID_EMAIL_ADDRESS:
+        'Please enter valid @education.gov.uk email addresses',
```

Comment:
Would probably re-word this message a little. For a start, we don't need to use 'please' as it implies there is a choice. There's an entire section advising on how to write error messages in the GOV.UK [Design System](https://design-system.service.gov.uk/components/error-message/#be-clear-and-concise) that's worth reading.

In this instance, I'd probably suggest something like 'Enter only @education.govuk email addresses'

Reply:
This is hilarious because I actually spent time reading articles about good error messages and being polite before writing these but totally missed the point about not using 'please'. I will remember to void errors in the imperative in future. Thanks.